src/config/topics.js

```diff
@@ -438,6 +439,17 @@ export const railPlus = {
   menu: <RailplusMenu />,
   embedded: true,
   overlaySide: 'left',
+  messageConfig: {
+    eventType: railplusPdfDownloadType,
+    callback: (evt) => {
+      if (
+        evt.type === railplusPdfDownloadType ||
+        evt.data === railplusPdfDownloadType
+      ) {
```

Comment:
you should  do this test in your MessageListener component. It will avoid adding it on each callback.

Reply:
done